### PR TITLE
Update attributes.md

### PR DESCRIPTION
**Pull request by AndreasDavour**
There's no explanation of what @property actually means, which makes the section hard to parse unless you follow the link at the bottom of the page to the reference section.

### CODE REVIEW

**Comment by Basile-z** [comment]
This is okay to merge unfortunately the CI is broken.

**Comment by ntrel** [comment]
Looks like there's trailing whitespace on these lines. Might be good to break long lines too.

**Comment by ZombineDev** [comment]
@ntrel Thanks for the reminder! The Netlify build and the trailing whitespace should now be fixed. As for splitting long lines - in general it's a good practice, but it's only strictly necessary for code snippets.
